garnbarn_api/serializer.py

```diff
@@ -28,12 +28,23 @@ def to_internal_value(self, value):
         return date_converted
 
 
+class ReminderTimeField(serializers.ListField):
+    def to_internal_value(self, data):
+        if data == []:
+            return None
+        data = super().to_internal_value(data)
+        data.sort()
+        return data
+
+
 class TagIdField(serializers.Field):
     def to_representation(self, value):
         return {
             "id": value.id,
             "name": value.name,
-            "color": value.color
+            "author": value.author,
```

Comment:
When we return the data to user, Actually we should return only `uid` and `name`  (The information that doesn't sensitive). If we return the entire object of `CustomUser`, It might be an API vulnerable 
 that can cause data leaks since anyone can know the other user's information by calling this API.